components/multiselect/multiselect.tsx

```diff
@@ -557,9 +557,10 @@ const paddedComponent = (WrappedComponent: any) => {
 const styles = {
     container: () => {
         return {
-            display: 'table-cell',
+            display: 'flex',
             verticalAlign: 'top',
-            width: '100%',
+            width: '271px',
```

Comment:
why setting a hard value here? Flex should handle the size

Reply:
After having a quick look these are the styles that need to be added:

```typescript
display: 'flex',
flex: 'auto',
overflow: 'hidden',
```

`overflow: 'hidden',` is the important part here, since it prevents the div from expanding! `width` can be removed completely since width is going to be handled by the parent flex container.